WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift

```diff
@@ -354,6 +354,7 @@ class ListTagProcessor: HtmlTagProcessor
 class AttachmentTagProcessor: HtmlTagProcessor
 {
     let textAttachmentIdentifier = "WPTEXTATTACHMENTIDENTIFIER"
+    static let attributeRegex = try! NSRegularExpression(pattern: "([a-zA-Z-]+)=(?:\"|')([^\"']+)(?:\"|')", options: .CaseInsensitive)
```

Comment:
Not really. I included the cap range more out of habit than anything.  Removing :)